rero_ils/modules/patrons/jsonschemas/patrons/patron-v0.0.1.json

```diff
@@ -121,6 +121,14 @@
           "patron"
         ]
       }
+    },
+    "communication_channel": {
+      "title": "Communication channel",
+      "type": "string"
+    },
+    "communication_language": {
+      "title": "Communication language",
+      "type": "string"
```

Comment:
we can limit the possible values by using an enum: https://json-schema.org/understanding-json-schema/reference/generic.html#enumerated-values

Reply:
done